comps/cores/mega/orchestrator.py

```diff
@@ -28,15 +28,13 @@
 
 
 class OrchestratorMetrics:
-    # Need an instance ID for metric prefix because:
-    # - Orchestror instances are not named
-    # - CI creates several orchestrator instances
```

Comment:
These are the reasons why this bug needed fixing, why they are removed?

(Only CI creates multiple Orchectrator, and therefore also multiple OrchestratorMetrics instances.)

Reply:
No, not only CI, check DocSum. https://github.com/opea-project/GenAIExamples/blob/main/DocSum/docsum.py#L137

Also even we do not have it in production, we should also remove the comment that contain "only CI needs" because initializing multiple Orchestrator instances is allowed not only for CI but for flexible usage in production.